src/SFA.DAS.Commitments.Api/Orchestrators/EmployerOrchestrator.cs

```diff
@@ -132,7 +132,8 @@ public async Task<long> CreateCommitment(long accountId, Commitment.CommitmentRe
                 Caller = new Caller { CallerType = CallerType.Employer, Id = accountId },
                 Commitment = commitment,
                 UserId = commitmentRequest.UserId,
-                Message = commitmentRequest.Message
+                Message = commitmentRequest.Message,
+                LastAction = (LastAction) commitmentRequest.LastAction
```

Comment:
If the 2 `LastAction` enums diverge then this cast won't work as expected any longer.

```suggestion
                LastAction = MapToDomainLastAction(commitmentRequest.LastAction)
```

Reply:
What are you proposing MapToDomainLastAction() does though?